### DISCUSSION TITLE
Extensions for (almost) inline javascript in WebView2

### DISCUSSION THREAD

**Discussion by ukandrewc** [MicrosoftEdge/WebView2Feedback#3531] [Tips and Tricks]
Hope it's OK to post this here, just thought it could be useful:

I've added a repository with extensions, that allows script to be composed in files, then executed as a file in debug, and as a resource in release.

It was inspired by trying to get as close as possible to inline editing of scripts, instead of creating strings or using resources that require a restart to refresh.

https://github.com/ukandrewc/WebView2.Winforms.JSHelper

**Comment by champnic** [comment]
This is awesome, thanks for sharing @ukandrewc!!
